src/components/SocialMediaLinks/SocialMediaLinks.mdx

```diff
@@ -0,0 +1,135 @@
+import {ArgTypes, Canvas, Description, Meta, Source} from "@storybook/blocks";
+import { GetTableData } from "./SocialMediaLinksUtils";
+import Table from "../Table/Table"
+import * as SocialMediaLinksStories from "./SocialMediaLinks.stories";
+import Link from "../Link/Link";
+
+<Meta of={SocialMediaLinksStories} />
+
+# SocialMediaLinks
+
+| Component Version | DS Version     |
+|-------------------|----------------|
+| Added             |  `Prerelease`  |
+| Latest            |  `Prerelease`  |
+
+## Table of Contents
+
+[//]: # ( @todo This TOC is pretty long. I don't see a DS component that allows you to create a collapsible list. Might be nice to have.)
+<details>
+    <summary>Click to Open</summary>
+- {<Link href="#overview" target="_self">Overview</Link>}
+- {<Link href="#with-controls" target="_self">With Controls</Link>}
+  - {<Link href="#available-social-media-types--their-default-values" target="_self">Available Social Media Types & Their Default Values</Link>}
```

Comment:
Technically, yes, this is a subsection of `With Controls`, but I don't think we need to show that heirarchy in the TOC.